network-mux/test/Test/Mux.hs

```diff
@@ -44,7 +44,7 @@ import           Control.Monad.Class.MonadThrow
 import           Control.Monad.Class.MonadTime
```

Comment:
I am not sure how to place a comment on the commit message, probably it's not possible. This commit: d163ea2f406c444e6623af57aa61796f5a961c35 has a typo `facter out` -> `factor out`. Not a big deal though.